docs/cellxgene_census_docsite_FAQ.md

```diff
@@ -132,3 +133,22 @@ You may get this error if you are trying to open a Census data build with an old
 
 If the error persists please file a [github issue](https://github.com/chanzuckerberg/cellxgene-census/issues/new/choose).
 
+## Why do I get an error when running `import cellxgene_census` on Databricks?
+
+This can occur if the `cellxgene_census` Python package is installed in a Databricks notebook using `%sh pip install cellxgene_census`. This command does _not_ restart the Python process after installing `cellxgene_census` and any pip package dependencies that were pre-installed by the Databricks Runtime environment but upgraded for `cellxgene_census` will not be reloaded with their new version. You may see `numba` or `pyarrow` related errors, for example.
+
+To fix, simply install using one of the following Databricks notebook "magic" commands:
+```
+pip install cellxgene-census
+```
+or
+```
+%pip install cellxgene-census
```

Comment:
I think this version still needs the  `-U` so I would adjust to be `%pip install -U cellxgene-census`

Reply:
My (questionable) understanding is that `-U` only upgrades the explicitly listed packages. Any transitive dependencies (e.g. numba) _will_ be upgraded _if_ the explicit packages are upgraded _and_ their dependencies require newer versions. So the transitive dependencies do get updated, as needed, even without `-U`.

I think the `-U` is only important if you're in the same notebook, a new cellxgene-census is released while it's active, and then you want to upgrade the cellxgene_census.

All that said, I'm happy to just include `-U` since my understanding may be wrong, and it's a good habit to include! 😄

Reply:
Oh interesting, when I tried `%pip install cellxgene-census` without the -U in my current notebook, it gave me an error

Reply:
Curious, what's the error? same one? I may learn something...

Reply:
Just tried re-running it and this time I didn't get an error, so I'm not sure now  😄